src/ping.rs

```diff
@@ -146,7 +146,7 @@ impl Pinger {
             let _size = sender
                 .send_to(&mut packet, &sock_addr.into())
                 .await
-                .expect("socket send packet error");
+                .ok();
```

Comment:
Maybe we can print trace log instead of just ignoring it or panic?

Reply:
You mean something like 067e7f0dc1764b5129b28d06ee5151f0e128a497?